### PR TITLE
include new pacakges

### PR DESCRIPTION
**Pull request by gustavoneves12**
include new packages in the package_sets:
- gui/vizkit3d_world
- gui/orogen/vizkit3d_world
- gui/orogen/underwater_camera_simulation

### CODE REVIEW

File: vizkit3d_world.autobuild [hunk 1]
```diff
@@ -0,0 +1,9 @@
+cmake_package 'gui/vizkit3d_world' do |pkg|
+	pkg.depends_on 'gui/robot_model'
```
Comment (doudou) [inline vizkit3d_world.autobuild]: Define the dependencies in the package's manifest.xml

Comment (gustavoneves12) [inline vizkit3d_world.autobuild]: Is it not necessary put dependency there?

Comment (doudou) [inline vizkit3d_world.autobuild]: It's not necessary, but that's the convention. We put dependencies in the autobuild files only for packages that don't have a manifest (i.e. packages that we don't develop such as e.g. gazebo or osg_ocean)

Comment (gustavoneves12) [inline vizkit3d_world.autobuild]: Ok. I got it.

Comment (doudou) [inline vizkit3d_world.autobuild]: I'll merge the PR when you have it done ;-)

**Comment by gustavoneves12** [comment]
I changed the cmake_package and put dependencies int he manifest.xml.
I performed an amend commit.

**Pull request merged**